orc-rt/docs/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+if (LLVM_ENABLE_SPHINX)
+  include(AddSphinxTarget)
+  if (SPHINX_FOUND)
+    if (${SPHINX_OUTPUT_HTML})
+      add_sphinx_target(html orc-rt)
+    endif()
+  endif()
```

Comment:
Should there be some kind of notice if docs aren't being built? If we're here, we requested that docs are built, but if `LLVM_ENABLE_SPHINX` is off, Sphinx isn't found or `SPHINX_OUTPUT_HTML` is off, we actually get no docs.

Reply:
I think this is ok: `LLVM_ENABLE_DOCS` controls both doxygen and sphinx docs. Configuring with `-DLLVM_BUILD_DOCS=On -DLLVM_ENABLE_SPHINX=Off` is "just build doxygen". If `-DLLVM_ENABLE_SPHINX=On` but the sphinx-build executable is missing then `add_sphinx_target` will error:
```
CMake Error at /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Failed to locate sphinx-build executable (missing: SPHINX_EXECUTABLE)
```

Reply:
Sounds good, thanks.